### PR TITLE
BaSP-TG-36: List employees Page

### PR DESCRIPTION
**Pull request by x-Ramses-x**
This functionality contains:
-Table list.
-Delete / Edit Icon.
-Delete feature.
-Redirect to Add form.
-Warning message for delete.

### CODE REVIEW

File: src/Components/Employees/index.js [hunk 1]
```diff
@@ -1,27 +1,43 @@
...
       <h2>Employees</h2>
       <div>
-        {employees.map((employee) => {
-          return <div key={employee.id}>{employee.name}</div>;
-        })}
+        <List list={list} deleteItem={deleteItem} setList={setList} />
       </div>
+      <button className={styles.button} href="/employees/form">
+        Add New Employee
+      </button>
```
Comment (CarVigil) [inline src/Components/Employees/index.js:38]: The <button> needs to be replaced with an <a> tag because the link doesn't work

**Review by CarVigil — CHANGES_REQUESTED** [review]
I

**Review by HiginiaMed — APPROVED** [review]
Good job!!

**Review by CarVigil — APPROVED** [review]

File: src/Components/Employees/ListItem/index.jsx [hunk 2]
```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import styles from './employees-item.module.css';
+
+let pencil = '✎';
+
+const ListItem = ({ listItem, deleteItem }) => {
+  const handleDelete = (_id) => {
+    deleteItem(_id);
+  };
+  console.log(listItem);
```
Comment (franco-marini) [inline src/Components/Employees/ListItem/index.jsx:10]: Here you have a console.log

Comment (franco-marini) [inline src/Components/Employees/ListItem/index.jsx:10]: Here you have a console.log, so you should remove it

File: src/Components/Employees/index.js [hunk 3]
```diff
@@ -1,27 +1,43 @@
...
-      .then((response) => response.json())
-      .then((response) => {
-        saveEmployees(response);
+  const deleteItem = (_id) => {
+    try {
+      const response = fetch(`${process.env.REACT_APP_API_URL}/employees/${_id}`, {
+        method: 'DELETE'
       });
-  }, []);
+      console.log(response);
```
Comment (franco-marini) [inline src/Components/Employees/index.js:22]: Here you have a console.log, so you should remove it

Comment (franco-marini) [inline src/Components/Employees/index.js:22]: Here you have a console.log, so you should remove it

**Review by franco-marini — CHANGES_REQUESTED** [review]
Nice work! Just review the minor corrections I left

**Review by franco-marini — APPROVED** [review]

**Pull request merged**